nrf_security/src/backend/nrf5x/entropy_nrf5x.c

```diff
@@ -8,8 +8,12 @@
 #include <entropy.h>
 #include <mbedtls/entropy.h>
 
-static_assert(defined(CONFIG_ENTROPY_GENERATOR), "CONFIG_ENTROPY_GENERATOR is not enabled.");
-static_assert(defined(CONFIG_ENTROPY_HAS_DRIVER), "CONFIG_ENTROPY_HAS_DRIVER is not set.");
+#if !defined(CONFIG_ENTROPY_GENERATOR)
+#error "CONFIG_ENTROPY_GENERATOR is not enabled."
+#endif
+#if !defined(CONFIG_ENTROPY_HAS_DRIVER)
+#error "CONFIG_ENTROPY_HAS_DRIVER is not set."
+#endif
```

Comment:
This looks like an upside-down testing.

I would prefer that we update the Kconfig and CMakeLists.txt file so that this file is only chosen if to be included using:
`zephyr_library_sources_ifdef(... entropy_nrf5x.c)

Reply:
Changed in https://github.com/NordicPlayground/nrfxlib/pull/99/commits/f6f1b80f881e09748c507735fe9834f3eb173bc8